backend/src/akvo/lumen/lib/auth.clj

```diff
@@ -133,47 +135,63 @@
                                   :visualisation-ids auth-visualisations
                                   :dashboard-ids auth-dashboards})))
 
+(defn- flow-check-permissions [flow-api request collector tenant data]
+  (let [t (prometheus/start-timer collector :app/flow-check-permissions {"tenant" tenant})
```

Comment:
Use https://xsc.github.io/iapetos/iapetos.core.html#var-with-duration

See https://github.com/akvo/akvo-unified-log/blob/3f12df3f01e85efa348ed5643d65b590e6c76526/src/akvo_unified_log/db.clj#L15 is you want to get fancy.